.github/dependabot.yml

```diff
@@ -6,6 +6,8 @@ updates:
     schedule:
       interval: "daily"
     labels:
-      - dependencies
-      - go
+      - automation
+    reviewers:
+      - "mtojek"
+      - "jsoriano"
```

Comment:
I'd prefer to put a [github] team here so we can more easily manage notifications and we don't need need to change this if the people related to the project changes. But we can go on like this by now.

Reply:
@v1v and I have discussed about this today and we have created a new "integrations-developer-experience" github subteam for this, so we can use it to assign reviewers.

@mtojek sounds this approach good to you?

Reply:
SGTM! I like the idea :)

Reply:
```suggestion
      - "elastic/integrations-developer-experience"
```